sphinx_gallery/tests/tinybuild/conf.py

```diff
@@ -16,6 +16,13 @@ def __call__(self, block, block_vars, gallery_conf):
         return matplotlib_scraper(block, block_vars, gallery_conf, **kwargs)
 
 
+def reset_argv(sphinx_gallery_conf, script_vars):
```

Comment:
This one, too

Reply:
Thanks, fixed it in aea11787f7d8e452ffe03169975810c79ae466aa